SmartSaver.MVC/Views/Transactions/Index.cshtml

```diff
@@ -32,39 +32,49 @@
                     <span aria-hidden="true">&times;</span>
                 </button>
             </div>
-            <form autocomplete="off" role="form" asp-controller="Transactions" asp-action="Create" method="POST">
+            <form id="form" autocomplete="off" role="form" asp-controller="Transactions" asp-action="Create" method="POST">
                 <div class="modal-body">
-                    <label>Amount</label>
-                    <div class="input-group">
-                        <input type="text" class="form-control" id="Amount" name="Amount"> <!-- Amount -->
-                        <div class="input-group-append">
-                            <span class="input-group-text">@currencySymbol</span>
+                    <div class="form-group">
+                        <label>Amount</label>
+                        <div class="input-group">
+                            <input type="text" class="form-control" id="Spending" name="Amount" />
+                            <div class="input-group-append">
+                                <span class="input-group-text form-group">@currencySymbol</span>
+                            </div>
                         </div>
+                        <span class="text-danger" id="SpendingStatus"></span>
                     </div>
-
-                    <label>Spending category</label>
-                    <select class="form-control" id="CategoryId" name="CategoryId">
-                        <!-- Category -->
-                        @foreach (var c in Model.Categories)
-                        {
-                            @if (!c.TypeOfIncome)
+                    <div class="form-group">
+                        <label>Spending category</label>
+                        <select class="form-control" id="CategoryId" name="CategoryId">
+                            <!-- Category -->
+                            @foreach (var c in Model.Categories)
                             {
-                                <option value="@c.Id">
-                                    @c.Title
-                                </option>
+                                @if (!c.TypeOfIncome)
+                                {
+                                    <option value="@c.Id">
+                                        @c.Title
+                                    </option>
+                                }
                             }
-                        }
-                    </select>
+                        </select>
+                    </div>
+                    <div class="form-group">
+                        <label>Choose date</label>
+                        <div class="date-input input-group">
+                            <input type="date" class="form-control">
+                        </div>
```

Comment:
Here I added date selction for user. I didn't make it work as I wanted to avoid the future pr conflicts